synapseAnnotations/data/experimentalData.json

```diff
@@ -171,11 +171,16 @@
         "description": "",
         "source": ""
       },
-      {
+       {
         "value": "MIB/MS",
         "description": "Kinomics assay using multiplexed kinase inhibitor beads and mass spectrometry (MIB/MS)",
         "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328787/"
       },
+      {
+        "value": "scCGIseq",
```

Comment:
I think this is fine. See above (MIB/MS) for example of specific sequencing method. I think with assays the more specific, the better (not an official contributing requirement AFAIK, though).